### PR TITLE
[Fix] Fix Dataset Display on README.md

### PR DESCRIPTION
**Pull request by MengzhangLI**
Fix some typos on readme.

(1) Display of dataset should be `###` not `####`. 

The `.yml` file would also be fixed by `pre-commit` command automatically.

(2) Add LoveDA dataset on cover `README.md` and  `README_zh-CN.md`

(3) Change dataset preparation link in  `README_zh-CN.md` from 

https://github.com/open-mmlab/mmsegmentation/blob/master/docs/dataset_prepare.md#cityscapes 

to

https://github.com/open-mmlab/mmsegmentation/blob/master/docs_zh-CN/dataset_prepare.md#cityscapes

(4) ADDED
Delete `pytorch 1.3.1` in `build.yml`

### CODE REVIEW

**Review by xvjiarui — APPROVED** [review]

**Comment by Junjun2016** [comment]
https://download.openmmlab.com/mmcv/dist/cpu/torch1.5.0/index.html
https://download.openmmlab.com/mmcv/dist/cpu/torch1.5.1/index.html

Should also change pt1.5.1 to pt1.5 for both cpu and gpu.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -204,7 +196,7 @@ jobs:
       - name: Install mmseg dependencies
         run: |
           python -V
-          python -m pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/${{matrix.mmcv_link}}/index.html
+          python -m pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch${{matrix.mmcv}}/index.html
```
Comment (Junjun2016) [inline .github/workflows/build.yml:199]:
```suggestion
          python -m pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch${{matrix.mmcv_link}}/index.html
```

**Comment by Junjun2016** [comment]
Hi @MengzhangLI 
Please fix the CI.

**Review by xvjiarui — APPROVED** [review]

**Comment by MengzhangLI** [comment]
Note: In this PR, the key `mmcv` is also modified. Which is actually PyTorch. 

It is referenced from this PR: https://github.com/open-mmlab/mmdetection/pull/6589/files.

**Comment by Junjun2016** [comment]
Should correct it.

**Pull request merged**